internal/utils.go

```diff
@@ -205,6 +205,7 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]interface{
 						len(policyTemplates)+1,
 						manifest["object-templates-raw"],
 						&policyConf.Manifests[i].ConfigurationPolicyOptions,
+						"",
```

Comment:
I think we still need to detect whether a name is provided alongside the `object-templates-raw` manifest and add it here.

Reply:
`object-templates-raw` is isPolicyTypeManifest = true so it should have `name` otherwise it throws an error  at `isPolicyTypeManifest, isOcmPolicy, err := isPolicyTypeManifest(` line 188. Do we need to add the name there?

Reply:
configurationPolicy name?

Reply:
The ConfigurationPolicy name is customizable in the generator manifest--it won't be set in the manifest in this scenario.

Reply:
@Dale I added a unit test for `object-templates-raw` When you are free, When you have a moment, could you review it to ensure we're on the same page?

Reply:
Looks good, thanks! I dug into this. It looks like tacking on the index of the policy-template is already the current behavior (for example, providing a ConfigurationPolicy directly would bring its name similar to how this feature does) and adding a counter like I'm proposing would be too different.